src/Controllers/Controller.php

```diff
@@ -12,9 +12,11 @@ abstract class Controller
     protected $db;
     private $loader;
 
-    //public function __construct(DBConnexion $db)
-    public function __construct($db)
+    public function __construct()
     {
+        if (session_status() === PHP_SESSION_NONE) {
+            session_start();
+        }
```

Comment:
You should put session start in your index.php. The controller shouldn't handle this stuff